Formula/dezoomify-rs.rb

```diff
@@ -0,0 +1,17 @@
+class DezoomifyRs < Formula
+  desc "Tiled image downloader"
+  homepage "https://github.com/lovasoa/dezoomify-rs"
+  url "https://github.com/lovasoa/dezoomify-rs/archive/refs/tags/v2.10.1.tar.gz"
+  sha256 "63ffceb01551fbbe920ee59f1b935f72ca55785edffc490efaedfbe1799ed8d2"
+  license "GPL-3.0-only"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    assert_match "USAGE", shell_output("#{bin}/dezoomify-rs --help")
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
You can download one as a resource. You're not limited to testing only using built-in test fixtures.